src/Symfony/Component/Translation/Translator.php

```diff
@@ -434,4 +444,14 @@ private function getConfigCacheFactory(): ConfigCacheFactoryInterface
 
         return $this->configCacheFactory;
     }
+
+    /**
+     * @param string $domain
+     *
+     * @return MessageFormatterInterface
```

Comment:
to be removed (and a real  return type be used instead)

Reply:
Thank you